### PR TITLE
 Vietnamese translation of Kubernetes Tutorials

### PR DESCRIPTION
**Pull request by tranquanghuy0801**
Translate the Explore Your App (Tutorials/Learn Kubernetes Basics) into the Vietnamese version.

### CODE REVIEW

**Comment by tranquanghuy0801** [comment]
I signed it

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Mục tiêu</h3>
+                <ul>
+                    <li>Tìm hiểu Kubernetes Pods là gì ?</li>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:24]: Please remove space before `?`. It should be `Tìm hiểu Kubernetes Pods là gì?`

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                    <li>Tìm hiểu Kubernetes Nodes là gì ?</li>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:25]: Please remove space before `?`

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                    <li>Khắc phục sự cố ứng dụng đã triển khai</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>Kubernetes Pods</h2>
+                <p>Khi bạn triển khai ứng dụng thông qua Module <a href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a>, Kubernetes tạo ra <b>Pod</b> để lưu trữ phiên bản chạy của ứng dụng của bạn. Một Pod là một khái niệm trừu tượng của Kubernetes, đại diện cho một nhóm gồm một hoặc nhiều ứng dụng containers (ví dụ như Docker hoặc rkt) và một số tài nguyên được chia sẻ cho các containers đó. Những tài nguyên đó bao gồm:</p>
+                <ul>
+                    <li>Lưu trữ được chia sẻ, dưới dạng Tập (Volumes)</li>
+                    <li>Kết nối mạng, như một cụm địa chỉ IP duy nhất</li>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:35]: `một cụm địa chỉ IP` -> `Cluster IP`
We don't need to translate some terms such as: Cluster, Node, Service, Pod,...

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                    <li>Thông tin về cách chạy từng container, chẳng hạn như phiên bản hình ảnh container hoặc các cổng cụ thể để sử dụng</li>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:36]: `hình ảnh container` -> `container image`

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                </ul>
+                <p>Một Pod mô phỏng một "máy chủ logic" dành riêng cho ứng dụng và có thể chứa các ứng dụng containers khác nhau được liên kết tương đối chặt chẽ. Ví dụ, một Pod có thể bao gồm cả container với ứng dụng Node.js của bạn cũng như một container khác cung cấp dữ liệu hiển thị bởi webserver của Nodejs. Các containers trong một Pod chia sẻ một địa chỉ IP và không gian cổng, chúng luôn được đặt cùng vị trí, cùng lên lịch trình, và chạy trong ngữ cảnh được chia sẻ trên cùng một Node.</p>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:38]: No need to translate `không gian cổng`

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+        <div class="row">
+            <div class="col-md-8">
+                <h2>Nodes</h2>
+                <p>Một Pod luôn chạy trên một <b>Node</b>. Một Node là một máy worker trong Kubernetes và có thể là máy ảo hoặc máy vật lý, tuỳ thuộc vào cụm. Mỗi Node được quản lí bởi Master. Một Node có thể chứa nhiều Pods và Kubernetes master tự động xử lí việc lên lịch trình các Pods thuộc các Nodes ở trong cụm. Việc lên lịch trình tự động của Master sẽ tính đến các tài nguyên có sẵn trên mỗi Node.</p>
+
+                <p>Mỗi Node ở Kubernetes chạy ít nhất:</p>
+                <ul>
+                    <li>Kubelet, một quy trình chịu trách nhiệm liên lạc giữa Kubernetes Master và Node; quản lí các Pods và các containers đang chạy trên cùng một máy.</li>
+                    <li>Một container runtime (như Docker, rkt) chịu trách nhiệm lấy hình ảnh container từ một nơi đăng kí (registry), giải nén container và chạy ứng dụng. Các containers chỉ nên được lên lịch trình cùng nhau trong một Pod duy nhất nếu chúng được liên kết chặt chẽ.</li>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:82]: `hình ảnh container` -> `container image`

File: content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+                </ul>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i> Các containers chỉ nên được lên lịch trình cùng nhau trong một Pod duy nhất nếu chúng được liên kết chặt chẽ và cần chia sẻ tài nguyên nhữ đĩa. </i></p>
```
Comment (truongnh1992) [inline content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html:88]: No need to translate `đĩa`

**Comment by tranquanghuy0801** [comment]
> Thank @tranquanghuy0801 for your great work.
> Please resolve my comments :-)

Thank @truongnh1992 for your suggestions, I have updated my work in the latest commit. Please have a check.

**Comment by truongnh1992** [comment]
/lgtm
/approve
Thank @tranquanghuy0801

**Pull request merged**